docs/src/main/sphinx/functions/objectid.rst

```diff
@@ -0,0 +1,49 @@
+===============================
+ObjectID function
+===============================
+
+.. _objectid:

```

Comment:
can you format the whole file to 80 char hard wrap width. Then I can proceed with more detailed input.

Reply:
Changed the name of ObjectID to MongoDB ObjectID and additional changes. Also changed the length of each line to meet the 80 character hard wrap width @mosabua